docs/reference/edot-sdks/android/index.md

```diff
@@ -0,0 +1,78 @@
+---
+navigation_title: EDOT Android
+description: The Elastic Distribution of OpenTelemetry Android (EDOT Android) is an APM agent based on OpenTelemetry. It provides built-in tools and configurations to make the OpenTelemetry SDK work with Elastic using as little code as possible while fully leveraging the combined forces of Elasticsearch and Kibana for your Android application.
+applies_to:
+  stack:
+  serverless:
+    observability:
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-sdk
+mapped_pages:
+  - https://www.elastic.co/guide/en/apm/agent/android/current/intro.html
+  - https://www.elastic.co/guide/en/apm/agent/android/current/index.html
+---
+
+# Elastic Distribution of OpenTelemetry Android
+
+The Elastic Distribution of OpenTelemetry Android (EDOT Android) is an APM agent based on [OpenTelemetry](https://opentelemetry.io/). EDOT Android provides built-in tools and configurations to make the [OpenTelemetry SDK](https://opentelemetry.io/docs/languages/java/) work with your {{stack}} using as little code as possible.
+
+All the features provided by the [OpenTelemetry SDK](https://github.com/open-telemetry/opentelemetry-java) are available for you to use and are pre-configured by the agent to work properly out of the box. This allows you to fulfill various use cases to better understand your app's performance.
+
+## Distributed tracing
+
+Distributed tracing allows you to see the full picture of how long your application has to wait for your backend services to provide a response, and why, as shown in the following example:
```

Comment:
```suggestion
Distributed tracing allows you to see the full picture of **how long your application has to wait for your backend services** to provide a response, and why, as shown in the following example:
```

This is worth emphasizing, as it's a common issue in android app development that doesn't have many solutions without using APM tools. So it's an important selling point.

Reply:
We don't really use bold for emphasis if we can avoid it.